internal/core/software_renderer.rs

```diff
@@ -1604,7 +1605,7 @@ impl<'a, T: ProcessScene> crate::item_rendering::ItemRenderer for SceneBuilder<'
         let geom = LogicalRect::from(size);
         if self.should_draw(&geom) {
             let mut border = rect.border_width();
-            let radius = rect.border_radius();
+            let radius = rect.logical_border_radius();
```

Comment:
In the master branch, the SoftwareRenderer now support screen rotation. (recently merged patch)
This mean you will need to rotate the corner (swap them)